src/test/java/com/epam/geometry/StringTo3PointsSieveTest.java

```diff
@@ -0,0 +1,74 @@
+package com.epam.geometry;
+
+import com.epam.geometry.data.DataException;
+import com.epam.geometry.logic.Logic;
+import com.epam.geometry.model.Triangle;
+import junit.framework.TestCase;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+public class StringTo3PointsSieveTest {
+    private StringTo3PointsSieve stringTo3PointsSieve;
+
+    @Before
+    public void prepare() {
+        stringTo3PointsSieve = new StringTo3PointsSieve();
+    }
+
+    @Test
+    public void testNotEnoughDataShouldThrowException() {
+        boolean thrown = false;
+        String notEnoughDataString = "1 2 45,67";
+        try {
```

Comment:
add exception in method

Reply:
Could you provide more details please?

Reply:
This is not my idea on how to test exceptions - I found it at Stackoverflow and just liked it. Is it a bad practice?